tests/fe/inf_fe_radial_test.C

```diff
@@ -16,9 +18,43 @@
 #include <vector>
 
 #include "libmesh_cppunit.h"
+#include "libmesh/dense_matrix.h"
+#include "libmesh/analytic_function.h" // for DenseVector
+#include "libmesh/fe.h"
 
 using namespace libMesh;
 
+bool system_solve_3x3(DenseMatrix<libMesh::Real> & A, DenseVector<libMesh::Real> & b, DenseVector<libMesh::Real> & x)
```

Comment:
And now `system_solve_3x3` is repeated twice... would definitely be good to avoid this.